ionic_user_interface/src/components/gym-selector/GymSelector.vue

```diff
@@ -152,12 +224,19 @@ export default defineComponent({
         gymLocationList.value = countryGymLocations;
         selectedCountryIso3.value = country.iso3;
         setUserCountry(country);
+        setNearestGym();
```

Comment:
Minor thing but i think need to handle error here?

Reply:
Actually I'll just remove this line as it seems a little out of place for the user flow